src/harness/fourslash.ts

```diff
@@ -1804,19 +1780,34 @@ namespace FourSlash {
             }
         }
 
+        /**
+         * @returns The number of characters added to the file as a result of the edits.
+         * May be negative.
+         */
         private applyEdits(fileName: string, edits: ts.TextChange[], isFormattingEdit = false): number {

```

Comment:
The `isFormattingEdit` parameter is omitted only once; consider making it non-optional. Same for the parameters in `printCurrentFileState`.